auth-service/src/main/kotlin/uk/gov/communities/delta/auth/Routing.kt

```diff
@@ -1,22 +1,32 @@
 package uk.gov.communities.delta.auth
 
+import io.ktor.http.*
+import io.ktor.http.content.*
 import io.ktor.server.application.*
 import io.ktor.server.auth.*
 import io.ktor.server.http.content.*
+import io.ktor.server.plugins.cachingheaders.*
 import io.ktor.server.plugins.ratelimit.*
 import io.ktor.server.response.*
 import io.ktor.server.routing.*
 import uk.gov.communities.delta.auth.controllers.external.DeltaLoginController
 import uk.gov.communities.delta.auth.controllers.internal.GenerateSAMLTokenController
 import uk.gov.communities.delta.auth.controllers.internal.OAuthTokenController
 import uk.gov.communities.delta.auth.controllers.internal.RefreshUserInfoController
+import uk.gov.communities.delta.auth.plugins.CSP
 import uk.gov.communities.delta.auth.plugins.addBearerSessionInfoToMDC
 import uk.gov.communities.delta.auth.plugins.addClientIdToMDC
 import uk.gov.communities.delta.auth.plugins.addServiceUserUsernameToMDC
 import uk.gov.communities.delta.auth.security.*
 
 fun Application.configureRouting(injection: Injection) {
     routing {
+        install(CachingHeaders) {
+            options { call, _ ->
+                if(call.response.headers["Cache-Control"] == null)
+                    CachingOptions(CacheControl.NoStore(CacheControl.Visibility.Private))
+                else null }
```

Comment:
Don't worry here, but for next time worth formatting files, you can either do them as you go, it's ctrl + alt + L in IntelliJ, or before you commit you can right click on a folder and "Refactor code", and that should give you an option to refactor all changed files.